pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/PinotFSSegmentUploader.java

```diff
@@ -54,6 +54,8 @@ public PinotFSSegmentUploader(String segmentStoreDirUri, int timeoutMillis) {
 
   public URI uploadSegment(File segmentFile, LLCSegmentName segmentName) {
     if (_segmentStoreUriStr == null || _segmentStoreUriStr.isEmpty()) {
+      LOGGER.error("Missing segment store uri. Failed to upload segment file {} for {}.", segmentFile.getName(),
```

Comment:
This missing log should suffice? You don't need the additional log above in the segment data manager?